CPAN/buildme.sh

```diff
@@ -1396,11 +1396,15 @@ function build_ffmpeg {
     if [ "$MACHINE" = "padre" ]; then
         FFOPTS="$FFOPTS --arch=sparc"
     fi
-    
+  
+    # Disable h264 on SunOS as there are relocation error when linking to Media::Scan. 
+    if [ "$OS" = "SunOS" ] && [ "$FFMPEG_PREFIX" = "ffmpeg-3.4.1" ]; then
```

Comment:
Once you make the requested change in the bindings/perl/Makefile.PL ('-W1,-shared' --> '-shared'), can you revert this to see if it links properly?